src/adapters/adbund.js

```diff
@@ -0,0 +1,66 @@
+var CONSTANTS = require('../constants.json');
+var utils = require('../utils.js');
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var adloader = require('../adloader');
+
+var adBundAdapter = function adBundAdapter() {
+  var timezone = (new Date()).getTimezoneOffset();
+  var bidAPIs = [
+    'http://us-east-engine.adbund.xyz/prebid/ad/get',
+    'http://us-west-engine.adbund.xyz/prebid/ad/get'
+  ];
+  //Based on the time zone to select the interface to the server
+  var bidAPI = bidAPIs[timezone < 0 ? 0 : 1];
+
+  function _stringify(param) {
+    var result = [];
+    var key;
+    for (key in param) {
+      if (param.hasOwnProperty(key)) {
+        result.push(key + '=' + encodeURIComponent(param[key]));
+      }
+    }
+    return result.join('&');
+  }
+
+  function _createCallback(bid) {
+    return function (data) {
+      var response;
+      if (data && data.cpm) {
+        response = bidfactory.createBid(CONSTANTS.STATUS.GOOD);
+        response.bidderCode = 'adbund';
+        Object.assign(response, data);
+      } else {
+        response = bidfactory.createBid(CONSTANTS.STATUS.NO_BID);
+        response.bidderCode = 'adbund';
+      }
+      bidmanager.addBidResponse(bid.placementCode, response);
+    };
+  }
+
+  function _requestBids(bid) {
+    var info = {
+      referrer: utils.getTopWindowUrl(),
+      domain: utils.getTopWindowLocation().hostname,
+      ua: window.navigator.userAgent
+    };
+    var param = Object.assign({}, bid.params, info);
+    param.sizes = JSON.stringify(param.sizes || bid.sizes);
```

Comment:
I am getting bid back for size [[300,250]] but not for [300,250]. Please add support for the same.